src/test/getAssetFromKV.ts

```diff
@@ -270,3 +270,73 @@ test('getAssetFromKV when namespace not bound fails', async t => {
   )
   t.is(error.status, 500)
 })
+
+test('getAssetFromKV when mimetype is html is not revalidated', async t => {
+  mockGlobal()
+  const event = getEvent(new Request('https://blah.com', {
+    headers: {
+      'if-none-match': 'blah'
+    }
+  }))
+  const res = await getAssetFromKV(event)
+  if (res) {
+    t.true(res.headers.get('content-type').includes('html'))
+    t.true(res.headers.has('etag') !== true)
+    t.true(res.status !== 304)
```

Comment:
Does the mock cache start empty for each `test()` block? If so, I'd expect this always to be true, since something has to exist in the cache for `getAssetFromKV()` to return a 304. Do we need to call `getAssetFromKV()` twice in this test?

Reply:
Because we're going to support HTML revalidation now, I've removed this test :)

Reply:
Test is now moot due to HTML support :)